src/config/graphical-session/gnome.md

```diff
@@ -0,0 +1,34 @@
+# GNOME
+
+## Pre-installation
+
+GNOME requires a display server such as [Xorg](./xorg.md) or a
+[Wayland](./wayland.md) compositor.
```

Comment:
Wouldn't it be more correct to say that "it can provide its own Wayland compositor (`mutter`)" ?